onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -227,6 +229,14 @@ Return Value:
                 this->ConvDepthwiseU8S8Kernel = MlasConvDepthwiseKernelAvx2<int8_t>;
                 this->ConvDepthwiseU8U8Kernel = MlasConvDepthwiseKernelAvx2<uint8_t>;
                 this->ComputeSumExpF32Kernel = MlasComputeSumExpF32KernelFma3;
+                
+                //
+                // Check if the processor supports Hybrid core architecture.
+                //
+
+                if ((Cpuid7[3] & 0x8000) != 0) {
```

Comment:
I've not looked at the flags in detail. Wondering if the check for hybrid core architecture can be done in one place and called from elsewhere?

Reply:
Do you expect to check for hybrid in other parts of the code as well?
For the current flow, I assumed all CPUID checks for MLAS can be kept under platform.cpp.
And here we already read Leaf 7 to check for AVX and I decided to piggyback on that read for hybrid check as well.